.travis.yml

```diff
@@ -1,7 +1,8 @@
 sudo: false
 language: python
 python:
-    - "2.7"
-    - "3.4"
+    # Align with the latest two LTS versions of Ubuntu
```

Comment:
Bionic has python 3.6, not 3.5.

https://packages.ubuntu.com/search?suite=bionic&searchon=names&keywords=python3